app/services/pending_payments.rb

```diff
@@ -0,0 +1,15 @@
+# Returns the capturable payment object for an order with balance due
+
+class PendingPayments
+  def initialize(order)
+    @order = order
+  end
+
+  def payment_object
+    @order.payments.select{ |p| p if p.state == 'checkout' }.first
```

Comment:
can we use meaningful and communicative var names, please :pray: ? we don't pay by char :trollface: